pkg/cvo/upgradeable.go

```diff
@@ -152,6 +153,31 @@ func (check *clusterOperatorsUpgradeable) Check() *configv1.ClusterOperatorStatu
 		return cond
 	}
 
+	// Set upgradeable = false to stop the y stream upgrades to happen when an upgrade is in progress.
+	// This will only impact the Y stream updates as CVO ignores this for z stream updates
+	var isUpgradeInProgress bool
+	for _, op := range ops {
+		if sc := resourcemerge.FindOperatorStatusCondition(op.Status.Conditions, configv1.OperatorProgressing); sc != nil {
```

Comment:
Will this only be true during an upgrade? I don't know but wondering if some thing else affecting a resource could cause it to enter this state. Perhaps that's okay and valid - i.e. if a resource is `OperatorProgressing` for any reason we want to block upgrades?

Reply:
> Will this only be true during an upgrade? I don't know but wondering if some thing else affecting a resource could cause it to enter this state. Perhaps that's okay and valid

Mostly yes , as per the API docs 
```
// Progressing indicates that the operator is actively rolling out new code,
// propagating config changes, or otherwise moving from one steady state to
// another.  Operators should not report progressing when they are reconciling
// a previously known state.
```

Reply:
your current code is checking "is there a Progressing condition?", which should pretty much always be true, even outside of updates when we are Progressing=False.  What you want is "is there a Progressing=True condition?", which would be:

```go
if sc := resourcemerge.FindOperatorStatusCondition(op.Status.Conditions, configv1.OperatorProgressing); sc != nil && sc.Status == configv1.ConditionTrue {
```